packages/service/src/ui-service/view/resource-models.ts

```diff
@@ -0,0 +1,3 @@
+export interface ResourceListResponse<T> {
```

Comment:
This seems out of place in /view/ since that's currently all about wrapping models in observable views (using MobX). Maybe we should have a top-level arm.ts or something, since this is about arm resources?

Reply:
Speaking of which, can we rename this ArmResourceListResponse?